packages/utils/src/invariant.ts

```diff
@@ -0,0 +1,22 @@
+const isProduction: boolean = process.env.NODE_ENV === 'production';
+// @TODO: allow consumer to set the prefix e.g `[ Text.Body ]`
+const prefix: string = 'Invariant failed';
+
+// Throw an error if the condition fails
+// Strip out error messages for production
+// > Not providing an inline default argument for message as the result is smaller
+const invariant = (condition: any, message?: string): asserts condition => {
+  if (condition) {
+    return;
+  }
+  // GIVEN product, supress the message.
+  if (isProduction) {
+    return;
+  }
```

Comment:
the original invariant throws on production
https://github.com/alexreardon/tiny-invariant/blob/master/src/tiny-invariant.ts#L18

Here we skip it.